src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+$primary-color: #060b35;
+$description-color:#616070;
+$button-color:#00acdc;
+
 body {
   margin: 0;
 }
+/*--region card--*/
+.card {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14px;
+  color: $primary-color;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  // margin: auto;
+  padding: 32px 16px 16px;
+
+  &-title {
+    display: none;
+  }
+
+  &__title {
+    display: none;
+  }
+
+  &__good {
+    position: relative;
+    width: 160px;
+    height: 134px;
+    left: calc(50% - 160px / 2);
+    margin: 0 auto 37px;
```

Comment:
position relative and left looks redundant here, try to add this rules to your card instead:
```
    display: flex;
    flex-direction: column;
    justify-content: center;
```